tests/test_coords.py

```diff
@@ -41,6 +41,14 @@ def test_featurecollection(self):
         self.assertEqual(pairs[0], (-115.11, 37.11))
         self.assertEqual(pairs[1], (-115.22, 37.22))
 
+    def test_geometrycollection(self):
+        p1 = geojson.Point((-115.11, 37.11))
+        p2 = geojson.Point((-115.22, 37.22))
```

Comment:
Hi @nnseva thank you for finding this bug and contributing a solution! I have one small request: can you make `p2` here a LineString and update the respective `assertEqual()` call to make this test more representative of a real-world heterogeneous GeometryCollection? If that passes then I will go ahead and merge it.